packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -224,9 +321,39 @@ export const Stackbox = () => {
       BigInt(balance.value) >= parseUnits(tokenAmount, fromToken.decimals)
     ) {
       setShowInsufficentBalanceError(false);
+      setIsStackConfirmed(true);
       openModal(ModalId.CONFIRM_STACK);
     }
-  };
+  }, [
+    balance,
+    endDateTime,
+    fromToken,
+    openModal,
+    setIsStackConfirmed,
+    startDateTime,
+    toToken,
+    tokenAmount,
+  ]);
+
+  useEffect(() => {
+    if (
+      isStackConfirmed &&
+      !isTokenListLoading &&
+      fromToken.symbol === balance?.symbol &&
+      !isRefetchingBalance &&
+      !isModalOpen(ModalId.CONFIRM_STACK)
+    ) {
+      openConfirmStack();
+    }
+  }, [
+    isStackConfirmed,
+    isRefetchingBalance,
+    isTokenListLoading,
+    fromToken.symbol,
+    balance?.symbol,
+    openConfirmStack,
+    isModalOpen,
```

Comment:
what's the idea with this hook?

Reply:
This hook is for opening the confirm stack when `stackConfirmed` query param is true. I honestly don't think we need this for now but it was suggested and wanted to try it out.

Reply:
Yes, I suggested and it's cool but not worth it right now. Does not work when no wallet connected and has other issues like increasing the complexity of the code.

Let's remove it.